### DISCUSSION TITLE
Not installing using altstore

### DISCUSSION THREAD

**Discussion by greendoescode** [qnblackcat/uYouPlus#917] [Q&A]
I have built my own ipa, just incase a fix was made between the last 2 weeks.
I've tried both using sources and directly installing the ipa, both to no evale.
Just stays stucking, loading, loading, loading withg the line near the end

https://github.com/GreenDiscord/uYouPlus/releases/download/untagged-4a54da74c7cb19a12836/uYouPlus_18.03.3_2.3.1.ipa my build if you want it.

**Comment by greendoescode** [comment]
Thought I should add, it also makes altstore univailable

**Comment by story1234** [comment]
I couldn't upgrade to 2.3.1.  It's stuck at 2.1 after using Altstore.  I tried yours and I tried to reinstall Altstore.

**Reply by greendoescode** [reply]
Thought I'd say, some how got it working
